src/solver/Main.java

```diff
@@ -1,7 +1,98 @@
 package solver;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Main {
+
+    static String getHelp() {
+        String helpMsg = "This program is intended to solve Linear Equations with any amount of variables using Gauss-Jordan Elimination. \n\n" +
```

Comment:
User StringBuilder (of course the compiler will use StringBuilder automatically, but the usage of it is more look like Builder pattern.